### ISSUE TITLE
508-defect-2 [COGNITION, SEMANTIC MARKUP]: Alert for cerner user should be designed and written semantically

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25049]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I want to navigate through logical headings that aren't skipped.
As a voice assistive tech user, I want links to be designed as links so I can activate them correctly.
As a keyboard user, I want links to be styled as links so that I know to use `enter` and not `space`.
As a user with cognitive disabilities, I want content to be designed simply in a way I can parse.

## Details

The alert for the cerner user has several issues to be addressed:
- heading skips from `h1` to `h3`
- some headings should be paragraphs or are unneeded
- links are styled as buttons

## Acceptance Criteria

- [ ] headings do not skip
- [ ] links are styled as links
- [ ] ~design is simplified (created #28395 to address this, once a path forward is determined, we'll likely create a separate ticket)~

## Steps to Recreate
- Login with the cerner test case user.
- Using headingsMap, confirm headings are skipped.
- Tab to the "go to..." links styled as buttons and confirm the `space` bar doesn't work

## Possible Solution 

Consider the following design:

- alert's `h3` is bumped up to an `h2`
- "our records show" is converted to a `p`
- "Chalmers..." is converted to a `strong`
- "Please choose..." is split up into a `h3` and `p`
- Links are styled as links

<img width="1792" alt="Screen Shot 2021-05-20 at 4 41 15 PM" src="https://user-images.githubusercontent.com/14154792/119046748-15aa3100-b98b-11eb-8625-58f0f5c19625.png">

Alternatively, if this content will be difficult for folks to understand, consider redesigning the UX around that. For example, providing a small form flow to help them decide which to pick may be helpful.

I'd recommend shortening the headings. Guidance from the content team @peggygannon would be valuable.

## Screenshots or Trace Logs
<img width="1792" alt="Screen Shot 2021-05-20 at 4 32 30 PM" src="https://user-images.githubusercontent.com/14154792/119047160-8e10f200-b98b-11eb-86d4-cd45f45efddb.png">
<img width="1459" alt="Screen Shot 2021-05-20 at 1 53 40 PM" src="https://user-images.githubusercontent.com/14154792/119047209-9c5f0e00-b98b-11eb-9816-36b152333236.png">

**Comment by tressaellen** [comment]
Hi @joshkimux - would you mind elaborating more on the 'design is simplified' acceptance criteria bullet? I'm happy to hop on a call if that's easier. Thank you!

**Comment by joshkimux** [comment]
@tressaellen Yes, more than happy to hop on a call! By "simplify" the design, I should have been more specific 🙇 
I meant to ask if it was possible to reduce the length and complexity of the content. Here's an example of how that _could_ be approached, but definitely is not the solution. It might be worth looping in Danielle to help?

![image](https://user-images.githubusercontent.com/14154792/128364969-95fa445c-62d4-41f1-8c76-5a6b03be5176.png)

**Comment by tressaellen** [comment]
@joshkimux thanks for the clarification! That makes sense. @ajakabcin do you mind making a content ticket for this? And looping in Danielle, beth and me?

**Comment by DanielleThierryUSDSVA** [comment]
Hi @ajakabcin @tressaellen @joshkimux! Had a few minutes to take a quick initial look at this, and I'm wondering if we can simplify this even more. Some initial versions below...would want to think about this more and check back on notes to make sure we're not missing something here in simplifying. Happy to also jump on a call to brainstorm together if that's helpful.

_# V1_

## Your care team may now use our new My VA Health portal

Choose your portal based on the facility for your appointment. 

You may need to disable your browser’s pop-up blocker to open the portal. If you’re prompted to sign in again, use the same account you used to sign in to VA.gov.

### For Charles P. Wylie Veterans Outpatient Clinic
[link] Use My VA Health

### For any other VA health facility
[link] Use My HealtheVet

_# V2_ 

## Your care team may now use our new My VA Health portal

Choose your portal based on the facility for your appointment:

- **For Charles P. Wylie Veterans Outpatient Clinic:** [link] Use My VA Health
- **For any other VA health facility:** [link] Use My HealtheVet

**Note:** You may need to disable your browser’s pop-up blocker to open the portal. If you’re prompted to sign in again, use the same account you used to sign in to VA.gov. 

_# V3_

## Choose your health management portal

Your care team may now use our new My VA Health portal. Choose your portal based on the facility for your appointment. 

You may need to disable your browser’s pop-up blocker to open the portal. If you’re prompted to sign in again, use the same account you used to sign in to VA.gov.

### For Charles P. Wylie Veterans Outpatient Clinic

[link] Use My VA Health

### For any other VA health facility

[link] Use My HealtheVet

_# V4_

## Choose your health management portal

Your care team may now use our new My VA Health portal. Choose your portal based on the facility for your appointment:

- **For Charles P. Wylie Veterans Outpatient Clinic:** [link] Use My VA Health
- **For any other VA health facility:** [link] Use My HealtheVet

**If you have trouble accessing the portal:** Make sure to disable your pop-up blocker. If you’re prompted to sign in again, use the same account you used to sign in to VA.gov.

**Comment by joshkimux** [comment]
I like version 4 for the clear language "if you have trouble accessing the portal." I also think headings _may_ be a bit too much for presenting options to the user that each only have one link and no other content.

Something short and sweet may be easiest for folks to parse through.

**Comment by ajakabcin** [comment]
If version 4 (noted [above](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25049#issuecomment-894440005)) works for you @tressaellen and @erikphansen then it's good for me as well!

**Comment by ajakabcin** [comment]
Thank you @joshkimux and @DanielleThierryUSDSVA for the feedback here! We'll go with version 4 and address in a separate ticket. I'll update the AC to reflect this.

**Comment by ajakabcin** [comment]
Given the changes in the content and design of the alert the initial two AC about headers jumping and appropriately styled links is no longer applicable. I'll close this as we already have another FE ticket to implement the revised content and design: #28575
